pkg/controller.v1beta1/suggestion/suggestion_controller_test.go

```diff
@@ -255,6 +246,84 @@ func TestReconcile(t *testing.T) {
 	// Test 4 - Update status condition for empty experiment
 	g.Expect(r.updateStatusCondition(&suggestionsv1beta1.Suggestion{}, oldS)).To(gomega.HaveOccurred())
 
+	// test 5 - Early Stopping Suggestion run
+	// Create early stopping config
+	jsonConfigEarlyStopping := map[string]katibconfig.EarlyStoppingConfig{
```

Comment:
Maybe we can add EarlyStopping in the Test 1, just extending [`katib-config`](https://github.com/kubeflow/katib/blob/9f6266084d357e664ee9080ae124e347bee4177f/pkg/controller.v1beta1/suggestion/suggestion_controller_test.go#L330) and [`Suggestion spec`](https://github.com/kubeflow/katib/blob/9f6266084d357e664ee9080ae124e347bee4177f/pkg/controller.v1beta1/suggestion/suggestion_controller_test.go#L319)?
Then, 1 test will covert: Deployment, Service and RBAC creation and we don't need to create another Suggestion in envtest ?
WDYT @tenzen-y ?

Reply:
It makes sense! @andreyvelich 
I'd like to fix it!